packages/vuetify/src/stylus/components/_dialogs.styl

```diff
@@ -84,6 +84,3 @@
       > .v-card__text
         overflow-y: auto
         backface-visibility: hidden
-
-  &--scrollable > form
```

Comment:
I believe by removing this that it will re-introduce #3713 based off of this https://github.com/vuetifyjs/vuetify/commit/f90c8b9

Reply:
^^ @MajesticPotatoe @KaelWD